docker/base/conf.d/docker.conf

```diff
@@ -0,0 +1,7 @@
+## When running Nuxeo in a Linux container, the JVM will automatically detect the cgroup memory limit with the
```

Comment:
Should this file be renamed 10-docker.conf? so it could be overridden easily

Reply:
Yes, even 00-docker.conf